src/settings.ts

```
@@ -45,6 +51,42 @@ export class AIChatSettingsTab extends PluginSettingTab {
+					await this.plugin.saveSettings();
+				}));
+
+		new Setting(containerEl)
+			.setName('Max Tokens')
+			.setDesc('Set the maximum number of tokens for the AI response')
+			.addText(text => text
+				.setPlaceholder('Enter max tokens')
+				.setValue(this.plugin.settings.maxTokens.toString())
+				.onChange(async (value) => {
+					const intValue = parseInt(value);
+					if (!isNaN(intValue) && intValue > 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)